File: demo/src/main/java/com/example/demo/models/Organization.java
package com.example.demo.models;

import lombok.*;

import javax.persistence.*;

@Data
@Entity
public class Organization {
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Id
    @Column(name = "Id")
    private int id;
    @Basic
    @Column(name = "Name")
    private String name;
    @Basic
    @Column(name = "MinVedCode")
    private Integer minVedCode;
    @Basic
    @Column(name = "ControlCode")
    private Integer controlCode;
    @ManyToOne
    @JoinColumn(name = "TerritoryId", referencedColumnName = "id")
    private Territory territory;
    @Basic
    @Column(name = "PrintingOrder")
    private Integer printingOrder;
}

File: demo/src/main/resources/static/js/entity_related/entity_descriptions/organization.js
import {ColumnDescription} from "./columnDescription.js";

export const entityDescription = [
    new ColumnDescription("ID", "id", "number"),
    new ColumnDescription("Название организации", "name", "string"),
    new ColumnDescription("Код мин./вед.", "minVedCode", "string"),
    new ColumnDescription("Код управления", "controlCode", "string"),
    new ColumnDescription("Территория", "territory.name", "string"),
    new ColumnDescription("Порядок печати", "printingOrder", "number"),
];